API Inventory restructure (#37090)
DataDog/documentation: content/en/security/application_security/api_posture/api_inventory/_index.md

---
title: API Inventory
description: Catalog API endpoints and services, and assess API security risk across your environment.
aliases:
  - /security/application_security/api-inventory/
further_reading:
- link: "https://www.datadoghq.com/blog/primary-risks-to-api-security/"
  tag: "Blog"

>   text: "Mitigate the primary risks to API security"

Reviewer: ```suggestion
  text: "Mitigate the primary API security risks"
```
I noticed that this blog title was different when I clicked on it, but I actually prefer your wording, so ignore me if the mismatch is intentional.

---

> [API Inventory][1] is a continuously updated catalog of the API endpoints and services discovered across your environment. It shows security context for each endpoint, such as authentication status, public exposure, sensitive data flows, and associated findings.

Reviewer: ```suggestion
[API Inventory][1] is a continuously updated catalog of the API endpoints and services API Posture discovers across your environment. It shows security context for each endpoint, such as authentication status, public exposure, sensitive data flows, and associated findings.
```
The passive voice seemed a little weird in an intro sentence. I think the verb reads a little better with a clear subject?

Inventory consists of two explorers:

> - **[API Endpoints][2]**: The API Endpoints explorer catalogs your individual endpoints, surfacing shadow APIs (undocumented endpoints with no API definition and not detected from Amazon API Gateway) and orphan APIs (documented endpoints without traffic), and helps you prioritize the endpoints most at risk.
> - **[Services][3]**: The Services explorer aggregates findings, vulnerabilities, and runtime signals by service, so you can assess each service's risk and security coverage.
> 
> To detect and respond to weaknesses, attacks, or misconfigurations on these endpoints, use [API Findings][4]. In the API Endpoints explorer, each row displays a findings chip that opens the related finding in API Findings.

Reviewer: ```suggestion
- **[API Endpoints][2]**: The API Endpoints explorer catalogs your individual endpoints, surfacing shadow APIs (undocumented endpoints with no API definition and not detected from Amazon API Gateway) and orphan APIs (documented endpoints without traffic), and helps you prioritize the endpoints most at risk. To detect and respond to weaknesses, attacks, or misconfigurations on these endpoints, use [API Findings][4]. In the API Endpoints explorer, each row displays a findings chip that opens the related finding in API Findings.
- **[Services][3]**: The Services explorer aggregates findings, vulnerabilities, and runtime signals by service, so you can assess each service's risk and security coverage.
```
Not sure if this last paragraph is applicable to both? Because it specifically refers to endpoints, I was wondering if grouping everything in the same bullet point here would make more sense.

Author: It applies to both, but I think the description could be revised to make that clearer.
